packages/core/src/browser/connection-status-service.ts

```diff
@@ -198,9 +198,8 @@ export class ApplicationConnectionStatusContribution extends DefaultFrontendAppl
     private statusbarId = 'connection-status';
 
     protected handleOnline(): void {
-        this.statusBar.setBackgroundColor(undefined);
```

Comment:
Can we introduce `setHoverBackgroundColor` instead of breaking APIs? We did not want to use class names in first place to limit capabilities for Theia extension developers.